masu/__init__.py

```diff
@@ -46,11 +46,15 @@ def create_app(test_config=None):
     app = Flask(__name__, instance_relative_config=True)
 
     # Load configs
-    if test_config is None:
-        app.config.from_object('masu.config.Config')
-    else:
+    if test_config:
         app.config.from_mapping(test_config)
 
+        # disable log messages less than CRITICAL when running unit tests.
+        logging.disable(logging.CRITICAL)
```

Comment:
I actually like having logging during tests.  We have one log now that I've been meaning to look into:

`ResourceWarning: unclosed <ssl.SSLSocket fd=31, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.1.23', 60938), raddr=('54.239.29.25', 443)>`

This most likely means we are doing something wonky in a test and it should probably be looked at.

Reply:
ok sounds good.  I just don't want to lose visibility of whats going on in our default settings just so dots print pretty during tests.